apps/api/src/campaign-file/campaign-file.controller.ts

```diff
@@ -37,7 +37,14 @@ export class CampaignFileController {
     }
     return await Promise.all(
       files.map((x) =>
-        this.campaignFileService.create(campaignId, x.originalname, x.mimetype, person, x.buffer),
+        this.campaignFileService.create(
+          campaignId,
+          'background',
```

Comment:
I think we should use an enumeration for this. One option can be background photo, another could be document or invoice. @kachar what do you think?

Another issue is that you are hardcoding the `'background'` now, when it should be coming from the front-end instead. When the user uploads a file he/she should decide what the role of that file is

Reply:
I do agree on both points :+1:

Reply:
@PepiPetrov Please use the new enum in this location so the hardcoded string is replaced